docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,132 @@
+
+For the purposes of this guide we'll create a Kubernetes service account with super-user cluster access (`cluster-admin`), however in a production environment you would want to exercise restrictions. Check out the guide on [Tiller and Role-Based Access Control](https://github.com/helm/helm/blob/master/docs/rbac.md) to learn more.
+
+Begin by creating a file named `rbac-tiller.yaml` with the following content:
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tiller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tiller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+```
+
+Create the service account using the `kubectl` command tool:
+
+```bash
+kubectl create -f rbac-tiller.yaml
+```
+
+Next, we use the `helm` client to install the Tiller server to the cluster:
+
+```bash
+helm init --service-account tiller
+```
+
+Voila, you have installed Helm to your cluster.
+
+## Step 3: Install Kubeapps
+
+Kubeapps is a open-source web-based UI for deploying and managing applications in Kubernetes clusters. It provides you a dashboard to:
+
+    -  Browse Helm charts from public or your own private chart repositories and deploy them to your cluster
+    - Upgrade, manage and delete the applications that are deployed in your cluster
+    - Browse and provision external services from the Service Catalog
+
+Kubeapps can be installed using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps) from the official [Bitnami charts repository](https://github.com/bitnami/charts).
+
+Begin by configuring the Helm client to use the Bitnami charts repository.
+
+```bash
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+Now you can easily Kubeapps to your Kubernetes cluster with the following command:
+
+```bash
+helm install \
+    --name kubeapps \
+    --namespace kubeapps \
+    --set mongodb.metrics.enabled=true \
+    bitnami/kubeapps
+```
+
+In addition to installing Kubeapps, the `mongodb.metrics.enabled=true` in the command enables the Prometheus exporter for MongoDB.
+
+> **Tip**:
+>
+> When installing applications to your cluster, remember to enable Prometheus exporters in applications that support them. The exported instrumentation data will be automatically scrapped by the Prometheus server installed in the cluster by BKPR and will provide useful insights into the clusters performance.
+
+Access to the Kubeapps dashboard installed using the above command is only possible using a proxy connection to the cluster. Please follow the instructions displayed in the output of the command for accessing the dashboard.
+
+However, if you would like the dashboard to be accessible externally over the Internet, you can do so with the following command:
+
+```bash
```

Comment:
Given we have functional TLS (with BKPR), is there a reason we don't _only_ suggest this install command?
(as in: is kubeapps auth insufficient for a public endpoint like this? If so, we should warn in the above somewhere; if not, we could just suggest using it by default?)

Reply:
I was being conservative to the idea of exposing the dashboard to the world. I've take your advice and made it the default now, while still pointing out how it could be disabled.